src/frontend/screens/Exchange/ExchangeSettingsBottomSheet.tsx

```diff
@@ -0,0 +1,210 @@
+import React, {useState} from 'react';
+import {View, StyleSheet, Pressable, ScrollView} from 'react-native';
+import {BottomSheetWrapper} from '../../sharedComponents/BottomSheetWrapper';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText';
+import {BodyText} from '../../sharedComponents/Text/BodyText';
+import OrangeStar from '../../images/OrangeStar.svg';
+import GreyLeaf from '../../images/GreyLeaf.svg';
+import {
+  useSetMediaSyncSetting,
+  useGetMediaSyncSetting,
+} from '../../hooks/server/mediaSync';
+import {MediaSyncSetting} from '../../sharedTypes';
+import {BLACK, DARK_GREY, NEW_DARK_GREY, WHITE} from '../../lib/styles';
+import {PrimaryButton, SecondaryButton} from '../../sharedComponents/Buttons';
+import {defineMessages, useIntl} from 'react-intl';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
+
+const m = defineMessages({
+  save: {
+    id: 'screens.ExchangeSettingsBottomSheet.save',
+    defaultMessage: 'Save',
+  },
+  close: {
+    id: 'screens.ExchangeSettingsBottomSheet.close',
+    defaultMessage: 'Close',
+  },
+  everythingTitle: {
+    id: 'screens.ExchangeSettingsBottomSheet.everythingTitle',
+    defaultMessage: 'Exchange Everything',
+  },
+  everythingDesc: {
+    id: 'screens.ExchangeSettingsBottomSheet.everythingDesc',
+    defaultMessage: 'Full size photos and audio. Uses more storage.',
+  },
+  previewsTitle: {
+    id: 'screens.ExchangeSettingsBottomSheet.previewsTitle',
+    defaultMessage: 'Exchange Previews Only',
+  },
+  previewsDesc: {
+    id: 'screens.ExchangeSettingsBottomSheet.previewsDesc',
+    defaultMessage: 'Reduced smaller size photos. No audio included.',
+  },
+});
+
+export const ExchangeSettingsBottomSheet = () => {
+  const {mutate: setMediaSyncSetting} = useSetMediaSyncSetting();
+  const currentSetting = useGetMediaSyncSetting();
+  const {formatMessage: t} = useIntl();
+  const [selected, setSelected] = useState<MediaSyncSetting>('everything');
+  const {goBack} = useNavigationFromRoot();
+
+  const handleSave = () => {
+    setMediaSyncSetting(selected);
+  };
+
+  React.useEffect(() => {
+    if (currentSetting) setSelected(currentSetting);
+  }, [currentSetting]);
+
+  return (
+    <BottomSheetWrapper>
+      <ScrollView contentContainerStyle={styles.container}>
+        <View style={styles.container}>
+          <OptionCard
+            setting="everything"
+            icon={<OrangeStar width={30} height={30} />}
+            title={t(m.everythingTitle)}
+            description={t(m.everythingDesc)}
+            selected={selected}
+            onSelect={setSelected}
+            testID="EXCHANGE.option-everything"
+          />
+          <OptionCard
+            setting="previews"
+            icon={<GreyLeaf width={30} height={30} />}
+            title={t(m.previewsTitle)}
+            description={t(m.previewsDesc)}
+            selected={selected}
```

Comment:
example of changes if using a `isSelected` prop as suggested elsewhere
```suggestion
            isSelected={selected === "everything"}
            onSelect={setSelected}
            testID="EXCHANGE.option-everything"
          />
          <OptionCard
            setting="previews"
            icon={<GreyLeaf width={30} height={30} />}
            title={t(m.previewsTitle)}
            description={t(m.previewsDesc)}
            isSelected={selected === "previews"}
```

Reply:
Yes. I did this...